css-canvas-medical/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="header">
+    <img class="logo" src="images/logo-blue.svg">
+    <a class="head-link">Product</a>
+    <a class="head-link">Implementation</a>
+    <a class="head-link">About Us</a>
+    <a class="head-link">Blog</a>
+    <button class="contact-button">Contact Us</button>
+    </div>
+    <div class="line"></div>
```

Comment:
Rather than having an element specifically for this, you should apply this styling to the `.header` element itself.